js/controllers/list/SelectListPanelItem.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<li class="pkpListPanelItem pkpListPanelItem--select" :class="{'--hasFocus': isFocused}">
```

Comment:
Sorry, I haven't dug into this stuff at all yet, but didn't we establish that starting classes off with a double-dash is a problem for Safari?

Reply:
You're right, I need to rebase against `master` with these changes in place.

Reply:
6cd4b3d0dca1cfc02b96175cdc40eef40f9df05b

Reply:
01df62e7bde1c79dc91992d9a1f28d75b042cc92